platform/persistence/platform-persistence-core-impl/src/main/java/org/codice/ddf/persistence/internal/PersistentStoreImpl.java

```diff
@@ -61,38 +65,43 @@ public PersistentStoreImpl(SolrClientFactoryImpl clientFactory) {
     }
 
     @Override
-    // Input Map is expected to have the suffixes on the key names
-    public void add(String type, Map<String, Object> properties) throws PersistenceException {
+    public void add(String type, Collection<Map<String, Object>> items) throws PersistenceException {
         LOGGER.debug("type = {}", type);
-        if (type == null || type.isEmpty()) {
+        if (StringUtils.isEmpty(type)) {
```

Comment:
❗️ These argument constraints should be mentioned in the JavaDoc comments of the interface.